src/hearings/models/hearings.enum.ts

```diff
@@ -305,6 +305,7 @@ export enum HearingResult {
   COMPLETED = 'COMPLETED',
   ADJOURNED = 'ADJOURNED',
   CANCELLED = 'CANCELLED',
+  AWAITING = 'AWAITING',
```

Comment:
is there any side effect to add this enum? As this seems to be used in the hearing result page.